src/chat/tools/getConnections.ts

```diff
@@ -36,6 +40,43 @@ export class GetConnectionsTool extends BaseLanguageModelTool<IGetConnectionsPar
   foundConnectionTypes: ConnectionType[] = [];
   missingConnectionTypes: ConnectionType[] = [];
 
+  prepareInvocation(
+    options: LanguageModelToolInvocationPrepareOptions<IGetConnectionsParameters>,
+    token: CancellationToken,
+  ): ProviderResult<PreparedToolInvocation> {
+    const { input } = options;
+    let invocationMessage: string;
+    let confirmationMessage: MarkdownString;
+    if (input.connectionType) {
+      invocationMessage = `Get all available ${getConnectionLabel(input.connectionType)} connections`;
+      confirmationMessage = new MarkdownString()
+        .appendMarkdown(`## ${getConnectionLabel(input.connectionType)} Connections\n`)
+        .appendMarkdown(
+          `This tool will look up all available connections of type **${titleCase(input.connectionType)}**.`,
+        )
+        .appendMarkdown(`Results will show the connection ID, name, and other details.`)
+        .appendMarkdown(`Do you want to proceed?`);
```

Comment:
I think, if we aren't using newlines, we may want to keep these all together in a single string with proper spacing since `.appendMarkdown` doesn't add any padding/spaces between them:
<img width="621" alt="image" src="https://github.com/user-attachments/assets/0ea4b3d0-ca31-456f-b5d3-7cd57a090212" />